manual/num/rotagrid.tex

```diff
@@ -59,3 +64,5 @@ \subsubsection{~Rotated grids} \label{sub:num_space_rotagrid}
 of a rotated grid model (NAEW).  Users may find more information in
 \emph{smc\_docs/Rotated\_Grid.pdf} or contact Jian-Guo Li for help
 (\url{Jian-Guo.Li@metoffice.gov.uk}).
+
+Note
```

Comment:
Extra lines left here?

Reply:
Oops. Yes - good spot! Fixed in latest commit.